multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml

```diff
@@ -23,6 +23,13 @@ metadata:
   name: antrea-mc-controller-role
   namespace: antrea-multicluster
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
```

Comment:
Could we limit the Namespace? Better to do so for other resources too @luolanzone

Reply:
Sure, I think we can limit the Namespace scope. I will check and refine this in a follow up PR.

Reply:
Ok.